db/seeds.rb

```diff
@@ -146,14 +254,42 @@
                             )
 product_desktop.save!
 
+product_printer = Product.create!(user_id: User.last.id, 
+                          title: "HP Printer", 
+                          description: "This HP printer is in great condition, used for 6 months only.", 
+                          status: "Pending", 
+                          condition: "Used (good)",category_id: category_printer.id, 
+                          address: "Alma Road, St.Albans AL1"
+                          )
+product_printer.photo.attach(io: URI.open('https://res.cloudinary.com/ds32iiece/image/upload/v1615281110/download_mpbrki.jpg'), 
+                            filename: 'printer.jpg', 
+                            content_type: 'image/jpg'
+                            )
+product_printer.save!
+
+product_monitor = Product.create!(user_id: User.last.id, 
+                          title: "Monitor 15inches", 
+                          description: "15inches Touch Screen LCD Monitor Display 1024x768 Resolution. It is 2 years old but used with care, minor scratches but in good usable condition.", 
+                          status: "Pending", 
+                          condition: "Used (good)",category_id: category_monitor.id, 
+                          address: "Winchester House, Merchant Street, Bow, Mile End, London E3"
+                          )
+product_monitor.photo.attach(io: URI.open('https://res.cloudinary.com/ds32iiece/image/upload/v1615281109/download_2_eqw6pt.jpg'), 
+                            filename: 'monitor.jpg', 
+                            content_type: 'image/jpg'
+                            )
+product_monitor.save!
+
 #Requests
 Request.create!(user_id: user_seeker_1.id, category_id: category_laptop.id)
-Request.create!(user_id: user_seeker_1.id, category_id: category_mobile.id)
-Request.create!(user_id: user_seeker_1.id, category_id: category_printer.id)
-Request.create!(user_id: user_seeker_2.id, category_id: category_desktop.id)
-Request.create!(user_id: user_seeker_2.id, category_id: category_monitor.id)
-Request.create!(user_id: user_seeker_2.id, category_id: category_printer.id)
-Request.create!(user_id: user_seeker_3.id, category_id: category_camera.id)
+Request.create!(user_id: user_seeker_2.id, category_id: category_laptop.id)
+Request.create!(user_id: user_seeker_5.id, category_id: category_mobile.id)
+Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id)
+Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id)
+Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id)
+Request.create!(user_id: user_seeker_6.id, category_id: category_printer.id)
```

Comment:
```suggestion
Request.create!(user_id: user_seeker_6.id, category_id: category_tablet.id, status: "Completed")
```

Reply:
There was booking for seeker_6 with the ipad, but not a printer, so I though we could change that here? The ipad booking was also accepted, so we need the request status to be "Completed."